clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -177,28 +175,24 @@ CompletableFuture<ClientResponse> sendAutoCommit(final Map<TopicPartition, Offse
                 });
     }
 
-    private class OffsetCommitRequestState {
+    private class OffsetCommitRequest {
```

Comment:
There's a bit of duplicated in naming, however, RequestState itself is a class, so that i'm calling this OffsetCommitRequest directly.  We can probably find a better way to name this.